src/main/java/seedu/address/model/calendarevent/DateTime.java

```diff
@@ -106,6 +106,12 @@ public boolean isAfter(DateTime other) {
         return this.localDateTime.isAfter(other.localDateTime);
     }
 
+    @Override
+    public int compareTo(DateTime other) {
+        assert(other instanceof DateTime);
```

Comment:
maybe we should throw exception instead of assert

Reply:
Hmm actually I think that may not even be needed. I don't see any exception for other compareTo or equals methods. So I'll just remove it.